src/renderers/shaders/ShaderChunk/bsdfs.glsl.js

```diff
@@ -268,17 +279,15 @@ vec3 BRDF_Specular_GGX_Environment( const in GeometricContext geometry, const in
 // http://www.jcgt.org/published/0008/01/03/
 void BRDF_Specular_Multiscattering_Environment( const in GeometricContext geometry, const in vec3 specularColor, const in float roughness, inout vec3 singleScatter, inout vec3 multiScatter ) {
 
-	float dotNV = saturate( dot( geometry.normal, geometry.viewDir ) );
+	float dotNV = max( 0.01, dot( geometry.normal, geometry.viewDir ) );
```

Comment:
Can you please explain your reason for this redefinition?

Reply:
Idea was this should never actually be 0, but that isn't true in some projections(?), and regardless doesn't change the outcome here. Will remove.